### PR TITLE
Add My Projects to dropdown

### LINKED ISSUE #1485: Add "My Projects" to wallet dropdown
## Description

It's hard to find the projects I own.

## Proposal

Add menu item to the wallet dropdown with a link to My Projects.

## Screenshots

Screen Shot 2022-07-22 at 5 28 51 PM

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Closes #1485. Adds 'My Projects' option to Menu dropdown. I picked the crown icon but let me know if there's a better one to use.

## Screenshots or screen recordings

image

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/Navbar/Wallet.tsx [hunk 1]
```diff
@@ -32,6 +34,17 @@ export default function Wallet({ userAddress }: { userAddress: string }) {
...
       </Menu.Item>
+      <Menu.Item
+        style={{
+          padding: menuItemPadding,
+          color: colors.text.primary,
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <Link href="projects?tab=myprojects">My Projects</Link>
```
Comment (tomquirk) [inline src/components/Navbar/Wallet.tsx:45]:
```suggestion
        <Link href="projects?tab=myprojects"><Trans>My projects</Trans></Link>
```

**Review by tomquirk — DISMISSED** [review]
one ting @jmilldotdev

**Review by tomquirk — APPROVED** [review]

**Comment by tomquirk** [comment]
Thank you @jmilldotdev ! cc @strath-m, you'll like this!

**Comment by strath-m** [comment]
Yes! I was literally just writing about this in my usability notes. Nice one @jmilldotdev

**Pull request merged**